### DISCUSSION TITLE
HA Integration - entities unavailable

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#4706] [General]
Folloing options are no longer working. Perhaps I didn't read a change I needed to make or this is HA issue. Can anyone suggest how I can fix it? I have already removed/re-added the frigate integration from HA and recreated both HA and Frigate containers.

<img width="331" alt="image" src="https://user-images.githubusercontent.com/7528001/207446094-8323b8f8-a429-4f25-bd96-d19b687b4f2e.png">

<img width="203" alt="image" src="https://user-images.githubusercontent.com/7528001/207446519-91f8a70c-df6e-4f3b-a7e7-f26a725699df.png">

**Comment by NickM-27** [comment]
Just click on them and enable them, they are only disabled by default but they work just fine

**Reply by bm55** [reply]
Thanks @NickM-27 ! 
From the entities section in HA, I was able to select and enable all Frigate entities that were disabled.
